### ISSUE TITLE
foreign characters not working 

### ISSUE THREAD

**Issue by juergenr** [tunnelvisionlabs/antlr4cs#163]
As I described in closed #159 the usage of foreign characters does not work (with the actual master).

Grammar:
```
grammar TestGrammar;
text:       WORD*;
WORD:       [a-zA-Z0-9öäüÖÄÜß我]+;
```
Code:
```
var errorListener = new ErrorListener();
var input = new AntlrInputStream("abcöäü我");

var lexer = new TestGrammarLexer(input);
lexer.RemoveErrorListeners();
lexer.AddErrorListener(errorListener);
var tokens = new CommonTokenStream(lexer);
var parser = new TestGrammarParser(tokens);
var tree = parser.text();
parser.RemoveErrorListeners();
parser.AddErrorListener(errorListener);
```
I get token recognition errors for each non ASCII character.

Is there something I missed?

**Comment by sharwell** [comment]
Hi @juergenr,

The issue was closed because it was a duplicate of another issue that was filed earlier (specifically #135).

The issue has been corrected in the latest working branch of the repository, but I hadn't uploaded a new release to NuGet yet. I just pushed release [4.5.4-beta001](https://www.nuget.org/packages/Antlr4/4.5.4-beta001) which will treat these files as UTF-8 by default instead of using the system encoding.

Sam
